content/docs/examples/advanced-egress/egress-gateway/index.md

```diff
@@ -669,7 +669,7 @@ The output should be the same as in the previous section.
     counter is:
```

Comment:
Add a reference to the code in line 667:
... of the `istio-proxy` egress gateway proxy ...

Reply:
Here the meaning is "the proxy of the egress gateway". What happened is just a change in the name istio/istio#7566 - the name of the proxy of the egress gateway changed from `egressgateway` to `istio-proxy`, it is still "the proxy of the egress gateway". The name of "the proxy of the egress gateway" is a minor detail, it is not relevant for the users who will just copy&paste that command to get the statistics of the proxy of the egress gateway.

Reply:
True, but it is relevant to the reader trying to understand the text and the sample code. I agree that it is a minor thing which is why I left it as a comment only. 😄